nodebuilder/node/cmds/node.go

```diff
@@ -1,32 +1,33 @@
-package main
+package cmd
 
 import (
-	"context"
 	"errors"
 	"strings"
 
 	"github.com/filecoin-project/go-jsonrpc/auth"
 	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/api/rpc/client"
+	"github.com/celestiaorg/celestia-node/cmd/celestia/util"
 )
 
+var rpcClient *client.Client
+
 func init() {
-	nodeCmd.AddCommand(nodeInfoCmd, logCmd, verifyCmd, authCmd)
-	rootCmd.AddCommand(nodeCmd)
+	Cmd.AddCommand(nodeInfoCmd, logCmd, verifyCmd, authCmd)
 }
 
-var nodeCmd = &cobra.Command{
+var Cmd = &cobra.Command{
 	Use:   "node [command]",
 	Short: "Allows administrating running node.",
 	Args:  cobra.NoArgs,
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-		rpcClient, err := newRPCClient(cmd.Context())
-		if err != nil {
-			return err
-		}
-
-		ctx := context.WithValue(cmd.Context(), rpcClientKey{}, rpcClient)
-		cmd.SetContext(ctx)
-		return nil
+		var err error
+		rpcClient, err = client.NewClient(cmd.Context(), client.RequestURL, "")
```

Comment:
super nitpicky, but i think it'd make the rest of the commands look a LOT nicer if the naming here with one of either:

`rpc := client.NewClient(`

or we alias the import to be rpc then use the naming

`client := rpc.NewClient`

i'm not sure what i prefer, but either over rpcClient as the client var.